specification/gp-connect-user-permissions.yaml

```diff
@@ -12,12 +12,14 @@ info:
     ## Overview
 
     Use this API to manage a patient's permissions to access the medical
-    record and services they can access at their GP practice.
+    record and services held at their GP practice.
 
     You can:
 
     - get a patient's permissions
+    - display only the sections that patients are permitted
```

Comment:
I agree there should be some info about what could be done with the API but I don't think it should be in this section. I see this section more about describing the API functions rather than what could be done with those functions.

Reply:
yes fair point, only added it in there because that's what they suggested in the design review